lib/internal/streams/from.js

```diff
@@ -30,7 +30,6 @@ function from(Readable, iterable, opts) {
     isAsync = true;
     iterator = iterable[SymbolAsyncIterator]();
   } else if (iterable && iterable[SymbolIterator]) {
-    isAsync = false;
     iterator = iterable[SymbolIterator]();
```

Comment:
This change creates an asymmetry between the two branches. I'd prefer removing the default assignment above the `if` statement.

Reply:
@tniessen: For the sake of asymmetry problem...Well, fixed ;)